pyproject.toml

```diff
@@ -1,20 +1,27 @@
-[tool.poetry]
-name = "geoh5py"
-version = "0.12.0-alpha.1"
+[build-system]
+requires = ["poetry-core>=1.0.0", "setuptools"]
+build-backend = "poetry.core.masonry.api"
 
-license = "LGPL-3.0-or-later"
+[project]
+name = "geoh5py"
 description = "Python API for geoh5, an open file format for geoscientific data"
-authors = ["Mira Geoscience <support@mirageoscience.com>"]
+license = "LGPL-3.0-or-later"
+keywords = ["geology", "geophysics", "data", "interoperability"]
+version = "0.12.0a1"
+dynamic = ["dependencies", "classifiers", "requires-python"]
+readme = "package.rst"
+authors = [{name = "Mira Geoscience", email = "support@mirageoscience.com"}]
+maintainers = [{name = "Matthieu Cedou", email = "matthieuc@mirageoscience.com"}, {name = "Dominique Fournier", email = "dominiquef@mirageoscience.com"}, {name = "Benjamin Kary", email = "benjamink@mirageoscience.com"}]
+
+[project.urls]
+homepage = "https://www.mirageoscience.com/mining-industry-software/python-integration/"
 repository = "https://github.com/MiraGeoscience/geoh5py"
-maintainers = [
-    "Benjamin Kary <benjamink@mirageoscience.com>",
-    "Dominique Fournier <dominiquef@mirageoscience.com>",
-    "Matthieu Cedou <matthieuc@mirageoscience.com>",
-]
 documentation = "https://mirageoscience-geoh5py.readthedocs-hosted.com/"
-homepage = "https://www.mirageoscience.com/mining-industry-software/python-integration/"
-readme = "package.rst"
-keywords = ["geology", "geophysics", "data", "interoperability"]
+
+[project.optional-dependencies]
+
+[tool.poetry]
+
```

Comment:
move `requires-poetry = ">=2.0"` here

Reply:
ok, changed it here: https://github.com/MiraGeoscience/geoh5py/pull/726/commits/54ebc4adf8e6eded5fe81f6f3ca7965409557a1c